test/meson.build

```diff
@@ -318,6 +342,11 @@ if build_machine.endian() != 'big'
     if zstd_dep.found() and zstd_dep.version().version_compare('<=1.3.3')
         check_sha = '437a0ec28def0c7f361402f6bb337cb963e6a3682bc742dd9e56171f6ece9881'
     endif
+    if host_machine.system() == 'windows'
+        split_at = '^<text:'
```

Comment:
Why do we have to preface this with a `^`?

Reply:
To escape `<` which is also the stream redirect operator (https://ss64.com/nt/syntax-esc.html).

On Windows `^` is similar to unix `\` escape char.